### DISCUSSION TITLE
I run device service"device-mqtt-go" by source code locally, but throw a panic at cache initialization, how can I solve this?

### DISCUSSION THREAD

**Discussion by cglin7** [edgexfoundry/community#157] [Q&A]
![image](https://github.com/edgexfoundry/community/assets/31946087/9b6320d7-5c63-472b-b0c8-35f57dbc08e6)

The version of source code is tag-V3.0.0

**Comment by lenny-goodell** [comment]
In 3.x you must provider flags for the service to know where to find the common config.
See https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedHybrid/#run-the-service-code-natively

The above panic has be fixed in `main` and replaced with an appropriate error message.
